apps/guide/content/docs/legacy/additional-info/proxy.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Using a proxy
+---
+
+It may be necessary to use a proxy when using Discord.js, for example when you are deploying your bot on a server that has a web filtering firewall that only allows outside traffic through the proxy. This guide will show you how to set up a proxy with Discord.js.
```

Comment:
```suggestion
It may be necessary to use a proxy when using discord.js, for example when you are deploying your bot on a server that has a web filtering firewall that only allows outside traffic through the proxy. This guide will show you how to set up a proxy with discord.js.
```